pkg/s3utils/s3utils.go

```diff
@@ -32,9 +32,9 @@ type Object struct {
 type sseType string
 
 const (
-	sseS3  sseType = "SSE-S3"
-	sseKms sseType = "SSE-KMS"
-	sseC   sseType = "SSE-C"
+	SseS3  sseType = "SSE-S3"
+	SseKms sseType = "SSE-KMS"
+	SseC   sseType = "SSE-C"
```

Comment:
What does `C` mean here?

Reply:
Custom key

Reply:
Then let us call it `SseCustomKey`. What do you think?

Reply:
modified .